daemon/src/main/scala/io/gearpump/cluster/local/LocalCluster.scala

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
why we need a new LocalCluster here? What's the difference between this local cluster and the local mode?